Expanded name list for Belgian ships

halcyon-gh opened the pull request:
Based on historical names found on http://www.marinebelge.be and extended with fictional names in the same vein.

halcyon-gh committed:
Merge pull request #2 from arkhometha/Development

Update 13/jul

halcyon-gh committed:
Extended list of Belgian ships

Partially based on historical names from http://www.marinebelge.be

halcyon-gh committed:
Repeat some names in different ship classes

halcyon-gh committed:
Move historical names to start of list

moretrim commented:
Here are my remarks after reviewing up to and including the frigates.

## French & Dutch spellings

- Henri***ë***tte needs a diaeresis in the Dutch spelling
- I would suggest that *Léopold I* be ‘Leopold I’ for consistency with other Victorian era royals for whom you mainly use the Dutch names.

  I would have suggested ‘Léopold Ier’ if you want to be using the French name (it should really be ‘Léopold I<sup>er</sup>’ but we can’t do that with Windows-1252), taking motivation from this modern day example: [French](https://fr.wikipedia.org/wiki/L%C3%A9opold_Ier_(navire)), [Dutch](https://nl.wikipedia.org/wiki/F930_Leopold_I_(1991)).

  However I cannot find any consistency in the matter and the Belgian navy refers to the ship in various ways in French<sup>[a](https://www.mil.be/fr/search/l%C3%A9opold?query=l%C3%A9opold), [b](https://www.mil.be/fr/search/l%C3%A9opold%20Ier?query=l%C3%A9opold%20Ier)</sup>, including ‘Léopold I’, ‘Léopold Ier” or ‘Léopold 1’. I’m also seeing the same trend for the 1855 ship, including in your source. This is really pissing me off and giving me a headache.
- Likewise *Léopold II* and ‘Leopold II’. The headache continues if we want the French name, because ‘Léopold II<sup>nd</sup>’ looks like nonsense to me so maybe Léopold I and Léopold II really are fine, geez.

## Queens & Princesses of Belgium

(plus a couple other female royals)

I could not figure out some of the namesakes, can you walk me through them?

- *Arcadie-Claret*: [this is the only person I could find](https://en.wikipedia.org/wiki/Arcadie_Claret), which has a couple issues
  * Claret was her family name, so it doesn’t follow the pattern
  * perhaps more pressingly, she was not a member of the royal family but a mistress of the king!
- *Sofie-Antoinette*, the closest I found is [Henriette Marie Charlotte Antoinette de Belgique](https://en.wikipedia.org/wiki/Princess_Henriette_of_Belgium) which should give the *Henriëtte-Marie*
- *Anna-Sophia*
- *Antoinette-Amalia*
- *Sophie-Theodora*
- *Ferdinande-Henriette*

## Dukes of Brabant

- Jan III seems to have been nicknamed ‘de Triomfator’, if you want to use that

## Prince-Bishops of Liège

- Even though I can find it in a couple of places, I’m not sure François-Antoine-Marie de Méan is the actual name in modern historiography whether in English or French. I would suggest instead [François-Antoine de Méan de Beaurieux](https://fr.wikipedia.org/wiki/Fran%C3%A7ois-Antoine_de_M%C3%A9an_de_Beaurieux). Hopefully it fits, because that’s a mouthful.

## Dukes of Lorraine

- typo in *Godfried van Boullion*, should be [Godfried van Bouillon](https://nl.wikipedia.org/wiki/Godfried_van_Bouillon)

moretrim commented:
All of the rest (also I missed a couple cruiser names the first time around):

# Cruiser order

The cruiser names have a slight anomaly: *Ambiorix* appears before *Jan Breydel*, unlike every other occurrence where it appears at the end of the *Jan Breydel* to *Jacob van Artevelde* sequence. Obviously this makes no actual difference, but a consistent order makes it easier to tell at a glance how the names were put together and to locate copy/paste errors when reviewing. Definitively not a big deal.

# Latin

What did inspire the Latin names? Putting a twist on the unmodded *Belgique*?

* *Flandria**e*** should likely be [Flandria](https://en.wiktionary.org/wiki/Flandria#Latin)
* *Hen**o**govia* should likely be Hen**e**govia ([Wikipedia](https://la.wikipedia.org/wiki/Hannonia) refers to [this source](https://books.google.fr/books?id=asE9AAAAcAAJ&pg=PA294&redir_esc=y&hl=en#v=onepage))
* I’m not 100% sure about Limburg, though I can track a couple things referring to it as Limburgum (e.g. [the marginalia here](https://www.google.fr/books/edition/M%C3%A9moires_sur_les_troubles_des_Pays_Bas/hvrQ6eUVVlYC?hl=en&gbpv=1&dq=limburgum&pg=PA228), plus a couple other occurrences in the same book)

# Rivers

The names taken from rivers are also mixed French & Dutch. Here I don’t mind as much when it comes to consistency as I do for royal family names. If (and I do mean ***if***) you want consistent names, I can help.

# Diacritics

* ***E**clair* should be [**É**clair](https://en.wiktionary.org/wiki/%C3%A9clair#Noun_2) (capitalisation never obviates the need for diacritics, even though that is something you’ll see frequently e.g. as in your source, [see this completely unrelated book](https://www.google.fr/books/edition/%C3%89clair_d_%C3%A9t%C3%A9/BMp1qpbhaVMC?hl=en&kptab=editions) which title is inconsistently spelled)
* same deal for ***E**meraude* -> **É**meraude

(Fun fact: commerce raider [*Ville de Fontaine-l'Évêque*](https://en.wikipedia.org/wiki/Fontaine-l%27%C3%89v%C3%AAque) gets it right.)

# Apostrophes

👍 The commerce raiders use the nice looking apostrophe (**’** U+2019 RIGHT SINGLE QUOTATION MARK, which despite the formal name is the recommended apostrophe), in the *Ville d’* pattern. However the boring straight apostrophe (' U+0027 APOSTROPHE) is also used in a couple other palces. I would suggest consistently using the one apostrophe for all (and why not pick the right looking one for the job).

* historical people names, e.g. *Jean d'Avesnes*
* *Ville de Fontaine-l'Évêque*

# City names

A tricky one to get right:

* *Ville de Le Roeulx* -> Ville du Rœulx (Ctrl-F on [this Wikipedia page](https://fr.wikipedia.org/wiki/Le_R%C5%93ulx) to see the phrase in action)

Since the ‘Ville de…’ pattern is French, I would suggest using all French names if you stick to it. I used this very handy [modern-day reference](http://pivot.tourismewallonie.be/index.php/9-pivot-gest-pc/214-liste-des-localites) for cross-reference purposes.

* *Poperinge* -> Poperinghe
* *Maaseik* -> [Maeseyck or Maseyk](https://fr.wikipedia.org/wiki/Maaseik), not used in the reference though I did find a [contemporary use of the former](https://www.google.fr/books/edition/Notice_historique_sur_la_ville_de_Maesey/RbA6AAAAcAAJ?hl=en)–I leave it up to you
* *Izegem* -> Iseghem

I don’t know Dutch well enough to suggest an alternative pattern, if you want to keep Dutch city names. [Stad Amsterdam](https://en.wikipedia.org/wiki/Stad_Amsterdam) looks like one, from the neighbours to the North.

# Ship classes

I did not review the ship classes, e.g. seeing what historical or historically-inspired names fit what Victoria 2 category. I liked a lot of it though! All in all I’m not super confident when it comes to my knowledge of warships during the period. If you still feel like the PR does need more eyeballs and ask me, I can look into it.

Out of the lot there is one thing that stands out to me: are *Aster* through *Primula* all taken from the modern-day minehunters? (That’s what your source lists.) If so, why did you use them for clipper & steam transports?

halcyon-gh commented:
Thanks for the extensive feedback @moretrim, much appreciated. I will update the patch request and address your remarks when I have some time again in the coming weeks.

halcyon-gh committed:
Use nice looking apostrophe

halcyon-gh committed:
Update list of feudal rulers

Hendrik IV -> removed ('infirm of mind and body')
Jan III -> Jan de Triomfator
Boudewijn III -> Boudewijn de Jonge
François-Antoine-Marie de Méan -> François-Antoine de Méan de Beaurieux
Fix typo Bouillon

halcyon-gh committed:
Use French names for women from the royal family

+ removed 'Arcadie-Claret'

halcyon-gh committed:
Accents and town names

halcyon-gh committed:
Add "Fin de la guerre"

https://en.wikipedia.org/wiki/Fin_de_la_guerre

halcyon-gh committed:
Replace modern flower class names by gemstones

+ append some clipper_transport names to the steam_transport list
+ change order of cruiser list

halcyon-gh committed:
Remove Latinised province names

halcyon-gh committed:
Repeat many names within light/heavy ship groups

halcyon-gh committed:
Remove names that don't fit in UI

Remove 2 last prince-bishops
Change "Boudewijn met de IJzeren Arm" to "De IJzeren Arm"

halcyon-gh commented:
In general I've applied most of the suggestions provided by @moretrim. The Latin province names which were taken from the original name list, but now removed. The modern minesweeper names were originally included to fill out the transports name list to ~30 names, but are now replaced by expanding the list of precious stones which are more in line with actually used names. The women's names for dreadnoughts also include the names of 6 (great-)grandmothers of Leopold I.

Leopold/Léopold I/II 1/2: Yeah, this can give a headache. Reading a bit further makes me believe the first ship was simply "Léopold", and "Léopold II" was the second ship named Léopold, not named after king Léopold the second.  The later ship "Léopold II" on the other hand probably was named for him. Because the names are used for cruisers, which are available only later in the game, it makes sense to me to use "Léopold I" to refer to the first king during the reign of the second. In short, I think I will keep the names "Léopold I" and "Léopold II".

Regarding matching names to ship classes: Belgium never had a significant navy before WW2, so I've interpreted the historical ship classes in the most positive light. Therefore "Louise-Marie" and "Duc de Brabant" end up as frigates even though they only had about half the amount of cannons of a typical frigate. Other sailing ships were included in the list of clipper transports, and ocean liners (paquebots) were included as steam transports. The later fishery protection ship "Ville d'Anvers" becomes a commerce raider, and the "Argus", "Minerva", and "Hirondelle" river patrol boats become monitors. Lastly, the post WW1 torpedo boats "Westdiep" and "Wielingen" become cruisers.

moretrim commented:
:+1: Top notch stuff, all well-organised. Can you check these two items:

- I believe *Sofie-Antoinette* should be Sophie-Antoinette when it comes to the French spelling ([this one](https://en.wikipedia.org/wiki/Princess_Sophie_Antoinette_of_Brunswick-Wolfenb%C3%BCttel), right?)
- *Ville de Le Rœulx* should be Ville du Rœulx (as in the title of [their official website](https://www.leroeulx.be/))

halcyon-gh committed:
Sofie -> Sophie & de le -> du
